### ISSUE TITLE
Error in latest 2.9.1

### ISSUE THREAD

**Issue by Kirancloudnix** [hybridauth/hybridauth#773]
`Fatal error: Class 'Facebook\Facebook' not found in /var/www/____________________________/hybridauth/Hybrid/Providers/Facebook.php on line 56`

Request someone to help

**Comment by ApacheEx** [comment]
@Kirancloudnix,

Hello, this PR should fix your problem #760. There is also similar issue here #754 - check it out.

**Comment by dereuromark** [comment]
This is still not fixed in 2.9.2
I now also have the fatal error and
```
Warning (2): require_once(/var/www/.../vendor/hybridauth/hybridauth/hybridauth/Hybrid/thirdparty/Facebook/autoload.php): failed to open stream: No such file or directory in [/var/www/.../vendor/hybridauth/hybridauth/hybridauth/Hybrid/Providers/Facebook.php, line 61]
```

Running on 2.9.2 of this lib.

**Comment by ApacheEx** [comment]
@dereuromark,
Of course it's not working, because you use library without composer install and even if you don't want use composer you have to download Facebook SDK and put in thirdparty directory. Please check comments from #760 (some documentation fixes coming soon)
